packages/react/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@stardust-ui/react",
+  "version": "0.20.0",
+  "description": "A themable React component library.",
+  "license": "MIT",
+  "author": "Levi Thomason <me@levithomason.com>",
+  "homepage": "https://github.com/stardust-ui/react#readme",
+  "bugs": {
+    "url": "https://github.com/stardust-ui/react/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://github.com/stardust-ui/react.git"
+  },
+  "jsnext:main": "dist/es/index.js",
+  "main": "dist/commonjs/index.js",
+  "module": "dist/es/index.js",
+  "types": "dist/es/index.d.ts",
+  "sideEffects": false,
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "gulp bundle --package react --umd"
+  },
+  "dependencies": {
+    "@stardust-ui/react-proptypes": "^0.20.0",
```

Comment:
just to be sure (although, pretty much confident you've testes `workspaces` feature) - is it the case that both `yarn install` and `lerna bootstrap` properly link inner dependencies?

Reply:
Yes, `yarn install` correctly links all dependencies. You can check it because `@stardust-ui/react-proptypes` is not published yet :)

Reply:
When I tried to change the package version (non-local):

![image](https://user-images.githubusercontent.com/14183168/52493218-95a19800-2bd3-11e9-8452-fccb47be0274.png)